src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_help.py

```diff
@@ -384,6 +384,39 @@
               --connector-name aci-connector --chart-url <CustomURL> --image-tag <VirtualKubeletImageTag>
 """
 
+helps['aks use-dev-spaces'] = """
+    type: command
+    short-summary: (PREVIEW) Use Azure Dev Spaces with a managed Kubernetes cluster.
+    parameters:
+        - name: --name -n
+          type: string
+          short-summary: Name of the managed cluster.
+        - name: --resource-group -g
+          type: string
+          short-summary: Name of resource group. You can configure the default group using 'az configure --defaults group=<name>'.
+        - name: --space -s
+          type: string
+          short-summary: Name of the dev space to use.
+        - name: --parent-space -p
+          type: string
+          short-summary: Name of a parent dev space to inherit from when creating a new dev space. By default, if there is already a single dev space with no parent, the new space inherits from this one.
```

Comment:
This short-summary at least needs to manually implement wrapping. Looking at other help documentation in az, it seems they have standardized on wrapping at 100 characters per line.